packages/api/app/Controllers/Http/TechnologyController.js

```diff
@@ -291,6 +293,7 @@ class TechnologyController {
 			throw error;
 		}
 		technology.likes = 0;
+		technology.status = 'draft';
 		this.indexToAlgolia(technology);
```

Comment:
Elas não deveriam ser indexadas ao algolia enquanto ainda estão como `draft`, não? Caso contrário já aparecerão nas pesquisas como se estivessem públicas :thinking:

Reply:
@mateus4k eu vou verificar isso nessa issue: https://github.com/ufersa/plataforma-sabia/issues/378

No momento essa função está indexando na hora do cadastro, futuramente teremos que muda pra quando ela for publicada.